algo/src/test/java/com/hartwig/serve/sources/ckb/curation/CkbCuratorTest.java

```diff
@@ -1,33 +1,108 @@
 package com.hartwig.serve.sources.ckb.curation;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
 import java.util.List;
 
 import com.google.common.collect.Lists;
 import com.hartwig.serve.ckb.datamodel.CkbEntry;
+import com.hartwig.serve.ckb.datamodel.clinicaltrial.ImmutableLocation;
+import com.hartwig.serve.ckb.datamodel.clinicaltrial.Location;
 import com.hartwig.serve.ckb.datamodel.variant.Variant;
 import com.hartwig.serve.sources.ckb.CkbTestFactory;
 
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
 import org.junit.Test;
 
 public class CkbCuratorTest {
 
+    private final CkbCurator curator = CkbCuratorTestFactory.createProperModel();
+
     @Test
     public void canCurateVariants() {
-        CkbCurator curator = new CkbCurator();
-
-        CurationEntry firstCurationKey = CurationFactory.VARIANT_MAPPINGS.keySet().iterator().next();
+        CkbVariantCurationEntry firstCurationKey = CkbVariantCurationFactory.VARIANT_MAPPINGS.keySet().iterator().next();
         CkbEntry entry = CkbTestFactory.createEntryWithGeneAndVariant(firstCurationKey.geneSymbol(), firstCurationKey.variant());
 
         List<CkbEntry> entries = curator.run(Lists.newArrayList(entry));
 
         Variant firstVariant = entries.get(0).variants().get(0);
 
-        CurationEntry firstCuratedValue = CurationFactory.VARIANT_MAPPINGS.get(firstCurationKey);
+        CkbVariantCurationEntry firstCuratedValue = CkbVariantCurationFactory.VARIANT_MAPPINGS.get(firstCurationKey);
         assertEquals(firstCuratedValue.geneSymbol(), firstVariant.gene().geneSymbol());
         assertEquals(firstCuratedValue.variant(), firstVariant.variant());
 
         curator.reportUnusedCurationEntries();
     }
+
+    @Test
+    public void canCurateFacilityNameBasedOnName() {
+        Location location1 = createLocation("Rotterdam", "Erasmus MC", "3062 PA");
+        Location location2 = createLocation("Rotterdam", "Erasmus ziekenhuis in Rotterdam", "3062 PA");
+        Location curatedLocation = createLocation("Rotterdam", "EMC", "3062 PA");
+
+        assertEquals(curator.curateFacilityName(location1), curatedLocation);
+        assertEquals(curator.curateFacilityName(location2), curatedLocation);
```

Comment:
`assertEquals` expects the expected first and then the actual, so you need to swap around the params!
(Side note: This is one reason why we use `assertThat` in ACTIN, to avoid this confusion!)

Also, I would rename `curatedLocation` to `expectedLocation` since the algo didn't actually curate that location, you create it manually!